invenio_rdm_records/records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -1,402 +1,712 @@
+          "format": "date-time"
         }
-      },
-      "extensions": {
-        "additionalProperties": {
-          "anyOf": [
-            {
-              "type": "array",
-              "items": {
-                "type": ["boolean", "number", "string"]
-              }
+      }
+    },
+
+    "provenance": {
+      "type": "object",
+      "description": "Record provenance.",
+      "additionalProperties": false,
+      "properties": {
+        "created_by": {"$ref": "#/definitions/agent"},
+        "on_behalf_of": {"$ref": "#/definitions/agent"},
+        "contact": {"$ref": "#/definitions/agent"}
+      }
+    },
+
+    "access": {
+      "type": "object",
+      "description": "Record access control and ownership.",
+      "additionalProperties": false,
+      "properties": {
+
+        "metadata": {
+          "description": "Metadata visibility (true - public, false - private)",
+          "type": "boolean"
+        },
+
+        "files": {
+          "description": "Files visibility (true - public, false - private)",
+          "type": "boolean"
+        },
+
+        "contact": {
+          "description": "Enable the contact feature to contact owners.",
+          "type": "boolean"
+        },
+
+        "owned_by": {
+          "description": "List of user IDs that are owners of the record.",
+          "type": "array",
+          "minItems": 1,
+          "uniqueItems": true,
+          "items": {"$ref": "#/definitions/agent"}
+        },
+
+        "embargo_date": {
+          "description": "Embargo date of record (ISO8601 formatted date time in UTC). At this time both metadata and files will be made public.",
+          "type": "string",
+          "format": "date-time"
+        },
+
+        "access_condition": {
+          "description": "Conditions under which access to files are granted",
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "condition": {
+              "type": "string",
+              "description": "Textual description under which conditions access is granted."
             },
-            {"type": "boolean"},
-            {"type": "number"},
-            {"type": "string"}
-          ]
+            "default_link_validity": {
+              "type": "integer",
+              "description": "Number of days"
+            }
+          }
         },
-        "description": "Configured additional metadata",
-        "type": "object"
-      },
-      "version": {
-        "description": "Record version tag.",
-        "type": "string"
+
+        "access_right": {
+          "default": "open",
+          "description": "TODO - Access right for record. - should be computed or moved to metadata?",
+          "type": "string"
+        }
+      }
+    },
+
+    "files": {
+      "type": "object",
+      "description": "Files associated with the record",
+      "additionalProperties": false,
+      "properties": {
+        "disabled": {
```

Comment:
I think using the "positive" version, i.e. `enabled: true|false`, might be more readable.